lib/tasks/images.js

```diff
@@ -85,11 +85,18 @@ module.exports = function(gulp, config) {
 			var transformStream = new stream.Transform({objectMode: true});
 			transformStream._transform = function (file, encoding, callback) {
 				if(file.imageTransform) {
-					sharp(file.contents).resize(file.imageTransform.width, file.imageTransform.height).withoutEnlargement().jpeg(file.imageTransform.options).toBuffer(function (err, buffer, info) {
-						if(err) console.error(err)
-						file.contents = buffer
-						callback(null, file)
-					})
+					jimp
+						.read(file.contents)
+						.then(function(image) {
+							image
+								.resize(file.imageTransform.width, file.imageTransform.height)
```

Comment:
[Sharp's resize](http://sharp.dimens.io/en/stable/api-resize/#resize) does crop. But JIMP just resizes it no matter what.

Reply:
Do you mean that it does scale up but sharp doesn't? Or how does the cropping work in sharp?

Reply:
No. If both height and width is specified it resized it (keeping aspect ratio) and then cropped it (to specified dimension). JIMP on the other hand resizes it but does not keep aspect ratio, which leads to distortion.

Reply:
Now I understand. Okay, I'll add the behavior.